### PR TITLE
history 5.3.2

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Add new entry for 5.3.2

# Testing this PR

Nothing to test
# Related reading

https://trello.com/c/AGJY6DBW/33-version-history-pr

cc @hflynn

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,27 @@
 OMERO version history
 =====================
 
+5.3.2 (May 2017)
+----------------
+
+This is a bug-fix.
```
Comment (hflynn) [inline history.txt]: "... bug-fix release."

File: history.txt [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+
+Improvements include:
+
+- improve populate_metadata plugin
+- fix deletion of a range of objects from CLI
+- allow to add textual annotation without namespace at import from CLI
```
Comment (hflynn) [inline history.txt]: textual annotations without a namespace can now be added at import using the CLI

File: history.txt [hunk 3]
```diff
@@ -1,6 +1,27 @@
...
+- allow administrator to move data between groups without being a group member from CLI
```
Comment (hflynn) [inline history.txt]: administrators can now use the CLI to move data between groups without being a member of those groups

File: history.txt [hunk 4]
```diff
@@ -1,6 +1,27 @@
...
+- expose more enumerations from ome-model
+- improve thumbnails retrieval in OMERO.web
+- add "Open With" option to right panel
```
Comment (hflynn) [inline history.txt]: ...in OMERO.web

File: history.txt [hunk 5]
```diff
@@ -1,6 +1,27 @@
...
+- prevent group owner from annotating other people data in OMERO.web
```
Comment (hflynn) [inline history.txt]: group owners are now prevented from annotating other people's data in OMERO.web

File: history.txt [hunk 6]
```diff
@@ -1,6 +1,27 @@
...
+- allow to preview a well, in that case the first field is viewed
```
Comment (hflynn) [inline history.txt]: This doesn't make very much sense to me, i had to go and look at the card to follow what you meant. Maybe:

"made the image preview in the righthand pane available for Wells, showing the first field by default"?

**Review by hflynn — COMMENTED** [review]
Also, should all be past tense to fit in with older versions (although this slipped for 5.3.1 while I was away) - fixed, added, etc

And I wonder if it would be neater if we had separate sections for the CLI and for OMERO.web so we don't have to add "in OMERO.web/using the CLI" to a bunch of individual points.

**Comment by hflynn** [comment]
Could you fix the tense of the 5.3.1 entry while you're editing the page anyway? And instead of 'allow to view/import" could we have "enabled viewing/importing" please?

File: history.txt (hunk 6 above)
Comment (jburel) [reply history.txt]: it was no longer possible to preview or click on full viewer when a well was selected
(double-click was still working)
This is now fixed

**Comment by jburel** [comment]
@hflynn I have added new entries for the coming PRs

**Comment by hflynn** [comment]
Looks good now, should be fine to merge once all green.

File: history.txt [hunk 7]
```diff
@@ -1,19 +1,51 @@
...
+- group owners are now prevented from annotating other people's data in OMERO.web
+- preview of wells now available in the right-hand panel
+
+Sysadmin changes include:
+
+- made the Django middleware classes configurable using a new property
+- added property to allow connections from specified origins (CORS)
+- administrators can move data between groups without belonging to source and target groups using the CLI
+- for OMERO.web apps to be available via "Open With" option, administrators need to use
+the "omero.web.open_with" configuration option
```
Comment (hflynn) [inline history.txt]: If this is a new line, you'll need to indent it to be part of the bullet point or the docs build will complain

**Comment by hflynn** [comment]
It was having 'using the CLI' at the end of the sentence that Mark and I were suggesting you fix for the data moving point, not the use of 'those groups'. Ideal wording would be 
"administrators can now use the CLI to move data between groups without belonging to those groups"

**Comment by jburel** [comment]
misunderstood

**Comment by hflynn** [comment]
👍 looks good

**Review by kennethgillen — COMMENTED** [review]
Nice to see the "open with" mentioned for SAs.

**Pull request merged**